packages/tinacms-authjs/src/index.ts

```diff
@@ -133,10 +133,46 @@ const TinaCredentialsProvider = ({
       authenticate(databaseClient, credentials.username, credentials.password),
   })
 
+const NextAuthAuthentication = ({
```

Comment:
Naming should be consistent with the other packages - AuthJS instead of NextAuth. Also thinking it would be good to disambiguate frontend (auth providers) from backend. Maybe call it <*>BackendAuthentication?